### PR TITLE
DBZ-2693 Added entry-point script for OCP environment

### PR DESCRIPTION
**Pull request by jcechace**
Adds entry-point script for OCP with the ability to execute scripts prior to any other db setup. 
Unforuntale DB2 in OCP seems to have issues to start when database directory in emptyDIr volume is not empty and for some reason that happens sometimes.

### CODE REVIEW

File: tutorial/debezium-db2-init/db2server/setup_db2_instance.sh [hunk 1]
```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
```
Comment (jpechane) [inline tutorial/debezium-db2-init/db2server/setup_db2_instance.sh:1]: @jcechace Is this file really needed? I don't see it used anywhere

Comment (jcechace) [reply tutorial/debezium-db2-init/db2server/setup_db2_instance.sh:1]: You are correct. That is the orignal from ibm image
